ee/clickhouse/sql/person.py

```diff
@@ -160,6 +154,62 @@
     table_name=PERSONS_DISTINCT_ID_TABLE, cluster=CLICKHOUSE_CLUSTER, database=CLICKHOUSE_DATABASE,
 )
 
+#
+# person_distinct_ids2 - new table!
+#
+
+PERSON_DISTINCT_ID2_TABLE = "person_distinct_id2"
+
+PERSON_DISTINCT_ID2_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER {cluster}
+(
+    team_id Int64,
+    distinct_id VARCHAR,
+    person_id UUID,
+    is_deleted Boolean,
```

Comment:
is this a user-defined type? do we even have access to them on our version?

Reply:
anyway, another comment came to mind: let's get rid of this? we won't populate it anyway and the merge tree won't care for it so it isn't even valid if we just want to have this here for safety.

Reply:
It seems to be a built-in type, used in PERSONS_TABLE_BASE_SQL as well. Works on cloud as well.

Re getting rid of this: let's chat. Some python codepaths I didn't update yet would make use of this!

Reply:
wut 
https://clickhouse.com/docs/en/sql-reference/data-types/boolean/

Reply:
Looking into it just seems to be an alias for `Int8`